homeassistant/components/bang_olufsen/media_player.py

```diff
@@ -129,6 +139,7 @@ def __init__(self, entry: ConfigEntry, client: MozartClient) -> None:
             serial_number=self._unique_id,
         )
         self._attr_unique_id = self._unique_id
+        self._attr_should_poll = True
```

Comment:
Why do we first set it to false, to only now set it to true?

Reply:
Yeah that makes no sense at this point. All other entities in the [HACS](https://github.com/bang-olufsen/bang_olufsen-hacs) version of the integration and the Event entities in https://github.com/home-assistant/core/pull/127550 are not polling, so I would say setting it to False makes sense by default as this is the only contradiction.